views/standard/spec/columns.js

```diff
@@ -1,4 +1,36 @@
 define(['jquery', 'appearanceUtils'], function($, utils) {
+
+   // Sorting functions that are run on each column.
+   var globalSorts = {
+      sortByMilestone: function(pull) {
+         var score = 0;
+
+         if (pull.milestone.title) {
+            var due_date = pull.milestone.due_on &&
+             new Date(pull.milestone.due_on);
+
+            // If milestone is past due date.
+            if (due_date && due_date < new Date()) {
+               score -= 1500;
+            } else {
+               score -= 750;
```

Comment:
They are just arbitrary numbers that are relative to other arbitrary numbers currently used in the other sorting functions. It would probably be a good idea to set a fixed scale of point values to choose from instead of just throwing some in that work.

I chose these in particular because it would override any other of the scores from the other sorts.

I have issue #17 open for suggestions for improving the sort feature. It would define a list of sorting functions (i.e. `is_mine`, `needs_cr`, `needs_qa`) that would be listed in the column's `sort` method. Then the sum of all the scores could be the pull's score for that column. That way there is a centralized number of sorting methods that a column can use.

Maybe we could include a multiplier for the score, like how CR and QA columns both factor in whether or not there is QA and CR needed, but the CR column could have a 5x multiplier on the `needs_cr` sort while the QA column could have a 5x multiplier on the `needs_qa` method.